### PR TITLE
Split out Stanford's LMS urls

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
With this, the only modification we have are the two lines at the end of the file.
Note: This will otherwise cause a conflict on the upcoming merge.

**Comment by stvstnfrd** [comment]
Ping: @kluo @caesar2164

File: openedx/stanford/lms/urls.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+urlpatterns = (
+    url(
+        r'^course_sneakpeek/{}/$'.format(
+            settings.COURSE_ID_PATTERN,
+        ),
+        'student.views.setup_sneakpeek',
+        name='course_sneakpeek',
+    ),
+    url(
```
Comment (caesar2164) [inline openedx/stanford/lms/urls.py]: is the change in ordering just alphabetization?

File: lms/urls.py [hunk 2]
```diff
@@ -459,6 +441,8 @@
         'lms.djangoapps.instructor.views.instructor_dashboard.instructor_dashboard_2',
         name='instructor_dashboard',
     ),
+
+
```
Comment (caesar2164) [inline lms/urls.py]: just undoing a forked line removal?

Comment (stvstnfrd) [reply lms/urls.py]: Yup.

File: openedx/stanford/lms/urls.py (hunk 1 above)
Comment (stvstnfrd) [reply openedx/stanford/lms/urls.py]: Yup.

**Comment by stvstnfrd** [comment]
retest this please

File: openedx/stanford/lms/urls.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        r'^get_analytics_answer_dist/',
+        'courseware.views.views.get_analytics_answer_dist',
+        name='get_analytics_answer_dist',
+    ),
+)
+if settings.SHIB_ONLY_SITE:
+    urlpatterns += (
```
Comment (kluo) [inline openedx/stanford/lms/urls.py]: These no longer function due to the FORCED_TPA_PROVIDER_ID effectively preventing the default logistration page from ever showing, perhaps we can just remove these if their use case is obsolete or can be achieved elsewise

Comment (kluo) [reply openedx/stanford/lms/urls.py]: Should this wait on #733

**Comment by stvstnfrd** [comment]
@kluo I pushed up a new commit to remove `SHIB_ONLY_SITE` from the LMS.

**Pull request merged**